commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/mapping/serialization/to/ToSubmission.kt

```diff
@@ -0,0 +1,39 @@
+package ebi.ac.uk.extended.mapping.serialization.to
+
+import ebi.ac.uk.extended.model.ExtSubmission
+import ebi.ac.uk.model.Submission
+import ebi.ac.uk.model.extensions.creationTime
+import ebi.ac.uk.model.extensions.modificationTime
+import ebi.ac.uk.model.extensions.releaseTime
+import ebi.ac.uk.model.extensions.secretKey
+
+/**
+ * Return a simple submission which do not contains submission secret information.
+ */
+fun ExtSubmission.toSimpleSubmission(): Submission {
+    return Submission(
+        accNo = accNo,
+        section = section.toSection(),
+        attributes = attributes.map { it.toAttribute() }.toMutableList(),
+        tags = tags.toMutableList()
+    )
+}
+
+/**
+ * Return a extended submission which contains submission secret information and internal information.
+ */
+fun ExtSubmission.toExtSubmission(): Submission {
+    val submission = Submission(
```

Comment:
This one is called `toExtSubmission` but returns a regular `Submission`. Is this intended?

Reply:
this function has two version one that includes public info (intended for file serialization and user api) and another that contains secret information (intended to be used by frontend) .

that is the reason that name do not match the return type. Any suggestion for naming?